### PR TITLE
Updates to TA's grade distribution graphs

### PR DESCRIPTION
**Pull request by brendahuang93**
## Ready for Review

This pull request adds all TAs' datasets into the ta grade distribution graph (which currently only displays a TA's dataset at a time). It also adds other visual features to the graph.

### Additions and Modifications:
* Changed dataset source to all TAs to show all TAs' grade distributions
* Increased graph size, added legends, and displays different colours for each TA's dataset
* Updated view summary, grader summary, and refresh graph to draw ta grade distribution graphs
* Fixed issue to allow different canvases to be drawn for each TA's grader summary graph
* Moved Grader Distribution below graph and removed the TAs' links under Grader Distribution

#### Ta grade distribution graph displays all TAs' grade distributions:
screen shot 2017-02-19 at 3 09 46 pm

#### Moved Grader Distribution below graph and removed TAs' links
screen shot 2017-02-19 at 3 10 03 pm

### CODE REVIEW

File: app/views/assignments/_grader_summary.html.erb [hunk 1]
```diff
@@ -8,20 +8,6 @@
                                              action: 'grader_summary',
                                              assignment_id: assignment.id }
   %>):</p>
```
Comment (david-yz-liu) [inline app/views/assignments/_grader_summary.html.erb]: You can get rid of the colon here, I think.

File: app/views/assignments/view_summary.js.erb [hunk 2]
```diff
@@ -1,14 +1,14 @@
 document.getElementById('assignment_summaries').innerHTML =
   "<%= escape_javascript(render partial: 'assignment_summary',
-                                locals: { assignment: @assignment, ta: @current_ta}).html_safe %>";
+                                locals: { assignment: @assignment, ta: @tas}).html_safe %>";
```
Comment (david-yz-liu) [inline app/views/assignments/view_summary.js.erb]: Change the local variable here to `tas` (from the singular `ta`) to better reflect the relevant value. Note that you need to change this in a couple of places (in all the relevant files).

File: app/views/graders/_ta_grade_distribution_graph.html.erb [hunk 3]
```diff
@@ -1,46 +1,63 @@
...
-    // Set up data
-    var data = {
-      // Set up labels [0, 5, ..., 95]
-      labels: Array.apply(null, Array(20)).map(function (_, i) { return i * 5; }),
-      datasets: [{
-        data: <%= ta.grade_distribution_array(assignment, 20) %>
-      }]
-    };
+    <% if ta.nil? || ta.first.nil? %>
+        <% @ta_id = 0 %>
```
Comment (david-yz-liu) [inline app/views/graders/_ta_grade_distribution_graph.html.erb]: The indentation in this file is off. You should use 2 spaces for indentation (everywhere).

Comment (david-yz-liu) [inline app/views/graders/_ta_grade_distribution_graph.html.erb]: Also, don't use `@` here (you can just name the variable `ta_id`).

File: app/views/graders/_ta_grade_distribution_graph.html.erb [hunk 4]
```diff
@@ -1,46 +1,63 @@
...
-    var options = {
-      tooltips: {
-        callbacks: {
-          title: function (tooltipItems) {
-            var baseNum = parseInt(tooltipItems[0].xLabel);
-            if (baseNum === 0) {
-              return '0-5';
-            } else {
-              return (baseNum + 1) + '-' + (baseNum + 5);
+    <canvas id='a<%= assignment.id %>_ta_<%= @ta_id %>' width='400' height='350'></canvas>
```
Comment (david-yz-liu) [inline app/views/graders/_ta_grade_distribution_graph.html.erb]: Try using a single `<%= ... %>` and Ruby string interpolation.

File: app/views/graders/_ta_grade_distribution_graph.html.erb [hunk 5]
```diff
@@ -1,46 +1,63 @@
...
+    <script>
+        //<![CDATA[
```
Comment (arkon) [inline app/views/graders/_ta_grade_distribution_graph.html.erb]: You can actually remove this, along with its corresponding ending tag.

**Pull request merged**